typescript/graphql-apollo-server/schema.graphql

```diff
@@ -5,7 +5,7 @@
 type Mutation {
   createDraft(authorEmail: String, content: String, title: String!): Post!
   deleteOnePost(where: PostWhereUniqueInput!): Post
-  publish(id: ID): Post
+  publish(id: Int): Post
```

Comment:
Do you know what these changes are about? The type of the Post.id did not change in the schema, so this confused me a bit when I encountered it.

Reply:
Oh that's fine, it should be `Int` not `ID` actually, that's a leftover from the past

Reply:
👍 Ok perfect, when beta is out you maybe have to explain to me how all this works in the first place.

Reply:
@nikolasburk - I had to manually change some types from ID to String to make the typescript/graphql-nextjs example actually work.

@jasonkuhrt - this might suggest that it was due to an issue with the example, and not an indication that we need to update nexus-prisma. Would be great if labs can confirm 🙏